doc/projects.md

```diff
@@ -63,6 +63,7 @@ Key                                  | Type      | Condition             | Defau
 `restricted.devices.usb`             | string    | -                     | `block`                   | Prevents use of devices of type `usb`
 `restricted.idmap.uid`               | string    | -                     | -                         | Specifies the allowed host UID ranges allowed in the instance `raw.idmap` setting.
 `restricted.idmap.gid`               | string    | -                     | -                         | Specifies the allowed host GID ranges allowed in the instance `raw.idmap` setting.
+`restricted.networks.access`         | string    | -                     | -                         | Comma-delimited list of network names that are allowed for use in this project. If not set all networks are accessible (depending on the `restricted.devices.nic` setting)
```

Comment:
```suggestion
`restricted.networks.access`         | string    | -                     | -                         | Comma-delimited list of network names that are allowed for use in this project. If not set, all networks are accessible (depending on the `restricted.devices.nic` setting).
```

Reply:
I've added the full stop for now anyway.

Reply:
The general rule is to add full-stops for complete sentences but not sentence fragments, but to be consistent within a table.
So ideally, we would make sure that all rows are either only fragments or full sentences (which is why I often move a second sentence into parentheses when I clean up a table).

If a table is inconsistent (and some rows have two sentences, for example), the rule is to use a full stop for all.

And what does definitely NOT work is having two sentences, but leave out the full stop for the second. ;)

Reply:
An alternative could be
```suggestion
`restricted.networks.access`         | string    | -                     | -                         | Comma-delimited list of network names that are allowed for use in this project (if not set, all networks are accessible, depending on the `restricted.devices.nic` setting)
```

Reply:
I dont mind either way so will leave as your original suggestion.